api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java

```diff
@@ -69,16 +69,16 @@ private DbInstitution newDbObject(final Institution modelObject) {
   private DbInstitution updateDbObject(
       final DbInstitution dbObject, final Institution modelObject) {
     return dbObject
-        .shortName(modelObject.getShortName())
-        .displayName(modelObject.getDisplayName())
-        .organizationTypeEnum(modelObject.getOrganizationTypeEnum())
-        .organizationTypeOtherText(modelObject.getOrganizationTypeOtherText())
-        .emailDomains(
+        .setShortName(modelObject.getShortName())
+        .setDisplayName(modelObject.getDisplayName())
+        .setOrganizationTypeEnum(modelObject.getOrganizationTypeEnum())
+        .setOrganizationTypeOtherText(modelObject.getOrganizationTypeOtherText())
+        .setEmailDomains(
             Optional.ofNullable(modelObject.getEmailDomains()).orElse(Collections.emptyList())
                 .stream()
                 .map(domain -> new DbInstitutionEmailDomain().setEmailDomain(domain))
                 .collect(Collectors.toSet()))
-        .emailAddresses(
+        .setEmailAddresses(
             Optional.ofNullable(modelObject.getEmailAddresses()).orElse(Collections.emptyList())
```

Comment:
This will also not be null but an empty collection.

Reply:
I don't think we can guarantee this because it's generated by Swagger with a null default:
`  private List<String> emailAddresses = null;`

Reply:
gotcha

Reply:
I had a proposal for API object factories to help with that, but it didn't make it far.